pkgs/development/php-packages/openswoole/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, stdenv, buildPecl, php, valgrind, pcre2 }:
+
+buildPecl {
+  pname = "openswoole";
+
+  version = "4.11.1";
+   sha256 = "sha256-Rhoa4ny86dwB3e86/1W30AlDGRUDYjK8RusquKF5Izg=";
+
+  buildInputs = [ pcre2 ] ++ lib.optionals (!stdenv.isDarwin) [ valgrind ];
+  internalDeps = lib.optionals (lib.versionOlder php.version "7.4") [ php.extensions.hash ];
+
+  doCheck = true;
+  checkTarget = "tests";
+
+  meta = with lib; {
+    changelog = "https://pecl.php.net/package/openswoole/${version}";
```

Comment:
Eval of this attribute currently results to: `"https://pecl.php.net/package/openswoole/22.05pre-git"` which is probably not desired.

The reason for this result is because `version` is never found from the package version because it's not a recursive attribute set or defined in a `let` block. The reason for it becoming that value is then because `lib.version` exists and the lib scope is inherited using `with lib;` for the `meta` attribute.

I see two paths to not make this a useless or bad value:
1. Put `version` as a `let version = version = "4.11.1"; in buildPecl { pname = "openswoole"; inherit version; ... ` and let the current line stay as it is.
2. Just drop this line because afaik its' not really used that much? Lots of packages is missing it.

Reply:
@etu I'm not an expert in Nix. I have taken the example from Swoole and other PHP packages. Happy to make changes to the PR as recommended.  

Would changing it to `changelog = "https://pecl.php.net/package/openswoole/4.11.1"; be acceptable or you prefer this line to be removed?

Reply:
Done as requested.